demos/vc_issuer/src/main.rs

```diff
@@ -293,6 +295,7 @@ async fn vc_consent_message(
     )
 }
 
+#[cfg(feature = "custom_origin")]
```

Comment:
thanks @przydatek ! I'm not very familiar with the issuer and its test suite. Is this something you can take over?

> We need the build without the custom origin only for FE testing, right?

we need both actually, since we test both with and without custom origin (and the design relies on the method existing or not on the canister to differentiate the cases)

Reply:
I can take over, but you're very close already (and have made the necessary changes to `build.sh` and added `vc_demo_issuer_custom_origin-build`-section) 😄, so the only changes needed from here are:
- rename the feature flag to `exclude_custom_origin`
- change the cfg-guards to `#[cfg(not(feature = "exclude_custom_origin))]`
- rename  `vc_demo_issuer_custom_origin-build` to `vc_demo_issuer_exclude_custom_origin-build` in canister-tests.yml
- undo changes to `vc-issuer-test:` in canister-tests.yml
- rename in `build.sh` the parameter `--custom-origin` to `--exclude-custom-origin` (and flip the logic)

This way you'll have the existing tests working, and a new wasm (that excludes custom origin) for FE-tests.

Reply:
ah, the right syntax for the cfg-guard seems to be `#[cfg_attr(not(feature = "exclude_custom_origin"), ignore)]` as you already use in line 621 below, sorry! 
(btw, with a change to `exclude_custom_origin` the line 621 won't be needed)